conneg-by-ap/profiles.ttl

```diff
@@ -0,0 +1,35 @@
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix prof: <http://www.w3.org/ns/dx/prof/> .
+@prefix role: <http://www.w3.org/ns/dx/prof/role/> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+
+@prefix cnpr: <http://www.w3.org/ns/dx/conneg/profile/> .
+
+<https://www.w3.org/TR/dx-prof-conneg/> a dct:Standard ;
+    rdfs:label "Content Negotiation by Profile" ;
```

Comment:
All literals should have an appropriate language tag

Reply:
Added @en to all long literals - not labels